### PR TITLE
UCT/RDMACM: fix ep name

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by alinask** [comment]
https://github.com/openucx/ucx/commit/78a005e419ac09b37283c26019fc1a04a74f3dd1 was overridden somehow..

File: src/uct/ib/rdmacm/rdmacm_cm_ep.h [hunk 1]
```diff
@@ -47,7 +47,7 @@ enum {
                                                    necessarily disconnected yet */
 };
 
-UCS_CLASS_DECLARE(uct_rdmacm_ep_t, const uct_ep_params_t *);
+UCS_CLASS_DECLARE(uct_rdmacm_cm_ep_t, const uct_ep_params_t *);
```
Comment (yosefe) [inline src/uct/ib/rdmacm/rdmacm_cm_ep.h]: why do we even need this CLASS_DECLARE?
it's not that any other class inherits from it
and UCX worked even when this line was wrong, no?

Comment (alinask) [reply src/uct/ib/rdmacm/rdmacm_cm_ep.h]: you mean why we are using classes to create/manage the ep?

Comment (yosefe) [reply src/uct/ib/rdmacm/rdmacm_cm_ep.h]: i mean UCS_CLASS_DEFINE in C file should be enough in this case

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 01-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/a32e8f0ff9415a93c2d1f9a8b7916d09038b45f9/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/c7989f59fbcb722c3ed03a6bc21e2e1d4f9c0893/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/d035d0f5fb2e0429050afb43bdf1338adea14e55/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/4290467fdcb206c4cb30f6dbecd5a88509adc5e7/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/82f0f1108e4dfc3621bd636ad966501ce44ed5c9/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/978a031c9adda352c59e9942a396ef2cc6d3db80/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/4f4ef3d1cdf3cb7e107edc94d1a911b5752ff5d1/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/1a869a5659824169e50de5083f34dcf886f31001/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/06735f70b6a10d60fd311a6cc4bdc55298291fe9/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/a73228908532b3a1237aadb4f733fc65e584cb82/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/8ee77d4aaf120dc40ae4637bd5254a53fce30d61/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/e8b03a2836e4e60d9aa206a1b6b11490b5637ad5/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/76a14cee4b576911dfac051865f5cd2bc433f16e/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/efbc308933e3567a45b1c1d0b0e11b914fc7479e/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/0015ff3bfcf86700f944c6b142e6e2a31161de07/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/34c75d4e8947b69a701a2db91861d94a3ea2d3a0/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/0fb7e9c186022eb8d2d0766a600882390b13fd2a/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/7f16bacdde819443ee154fe97b16edf684b47d27/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/5d9eb1b6bc2f6f493b5a2e571fb30f35fa7da0d6/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/16038dd40a64c257111952beac7de7fd88398da9/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/d868f4386ecf4c3bea6c4530c54456c6684e0850/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/a28933276eda5bda9b8662274e103a175f032023/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/181788cc9bbf9161b80557bdab53110d45245022/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/35df855a36a7aab2ea49bdc85cb7f40e0e298a3b/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/283efdb19fde351669a7383b1ebff540/raw/8c9402646822b858bdcb0c51791a434e6d587925/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 02-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/2642ff3a77a524d04775ccbd6b69e2b3438d6464/_main) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/de9abca28dabb79e3cb9f08deac0ca005c038ebd/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/c0d203efe949ec624e29c2315417eb153461837e/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/07495ddb8ff14e0b79ba47e7d55353bcacc62a5b/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/dadcf32ba27cc34ca3868ab9b43fe7527ba898b8/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/fc3865191667f3668e6e946cdcd8fd1c490a8b9d/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/cf566cb70accf6183cd667057faaba4a24551557/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/d6420269af87e369c45dd9302e8394a6234df299/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/592ad9e50f38c3bda6f9b62e532a13c927f8f7b8/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/00c702826a50a2e0e17f4f5ae1104c5c685a67b8/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/ba3f09d1792edb5348b4cd068a6d0f67047be93a/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/46ba96022ae239d6bef42703aaff8f3c404a9194/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/92a7ba7ae33dc83ed422891ce68221dacf5d497e/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/ceb1910d8e16f3b5e65e551cc3f29397235d37bb/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/1698a1e9091c8d7a1b6f1d1d3bbc6f732229572b/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/968357338c6d10ec9fafc0c6877a4d7a9d3bcab2/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/4f26bc849c339d9988c4c3ff5d765d0c568e36b2/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/63f60388c3d83a13e5c8f8622f693597b6801702/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/49ba3936b9d87a6cda6fa516db25320900d94af5/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/cc343086d3eb14bdd1cbab15f0fff7d6217d6a5c/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/28235f9929e0ef22b3462538f5f259fa03ab66d8/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/be5c88e59eca3eaaa0a96f046db789dee8123d91/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/26b505e1feba8fd3804103107690edc70265d8de/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/ef4d7714397e580b733871375a0fe38424620f59/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/eb40611754d6575a2066bf4e596062f1/raw/448fa58e74e4517b0b8866e73dc91fc2fd97cf51/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by alinask** [comment]
bot:mlx:retest

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 02-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/b2d9d1d38d9df52f6f79784ac6436b6c23fdac53/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/44416f21fcb7f73caf4c91a4b51d603588273c04/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/f581cf11bc6bf9ba8c3825591104da3771f234e9/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/fdb69d04dd6d32241fb48ab3bbe1e10a73b08443/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/862be8ce426f0604dc3f3e33be979662424c5ea0/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/b4aae3186d923d0bd132b8087ef340596ca391cc/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/4942b44fd583da0884399b661cf55dfeea0839d7/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/cdcf38b9b5d590f91c51ed927322a23737615781/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/ed7ce582610f64ded3bb406c0ef128b1a7f60500/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/d92d54d487bb04ab1bb5d5a1cdca25842c3b0ed6/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/c0a629eefafe4c87f623934982d41a8f14043850/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/c98a7c3b28b6178a6f54231be918819d020f2846/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/c2cf13b2b17813050e4c3d71c8baaa66bdf5ca68/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/8454ba966f11d97dd7b236fca34154961b7d2176/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/2c0bbd1d2708cabfaed9d9b559ba90d74b917774/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/3a224918bc070bfae461a08aead5a7cac3af19c4/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/346e7e7584ebecd2d0f347b308541cea51d3a9a2/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/340cafb80c952cdcb6e62e3dc16d9c098d9eca84/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/c819783e21205f8220254f9a42da98444349b772/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/4e10f7cdc18037578459b4b2986dac0ac012fd20/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/88f23b5ca702c73425d43f60339b4fc460385237/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/4e9917cb53835605407d08265cbc80e035302e28/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/4d998cd08a896bb396ceca937da9bbbc490050ed/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/e7bd7b7a918b78c9b2bfd31763e19dfa6d2a1fda/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/983e5a26f115c588d2e39f3f98ad5331/raw/ebddf5399ede35891020b5687e5401bfbea921ef/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**